biggus/__init__.py

```diff
@@ -2764,15 +2801,23 @@ def shape(self):
 
     @property
     def sources(self):
-        return (self._array1, self._array2)
+        if self._array2 is None:
+            result = (self._array1, )
```

Comment:
This line is either unfinished or you've got trailing whitespace inside your one-length tuple...